game/tf2classic/scripts/items/items_game.txt

```diff
@@ -7778,7 +7935,422 @@
+		}
+		"701"
+		{
+			"name"	"duck rating"
+			"attribute_class"	"duck_rating"
+			"description_string"	"#Attrib_duck_rating"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"positive"
+			"stored_as_integer"	"1"
+		}
+		"702"
+		{
+			"name"	"duck badge level"
+			"attribute_class"	"duck_badge_level"
+			"description_string"	"#Attrib_duck_badge_level"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"positive"
+			"stored_as_integer"	"1"
+		}
+		"703"
+		{
+			"name"	"tag__eotlearlysupport"
+			"attribute_class"	"tag__eotlearlysupport"
+			"description_string"	"#Attrib_eotl_early_supporter"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"positive"
+			"stored_as_integer"	"1"
+		}
+		"704"
+		{
+			"name"	"unlimited quantity hidden"
+			"attribute_class"	"unlimited_quantity"
+			"description_format"	"value_is_additive"
+			"hidden"	"1"
+			"effect_type"	"positive"
+			"stored_as_integer"	"0"
+		}
+		"705"
+		{
+			"name"	"duckstreaks active"
+			"attribute_class"	"duckstreaks_active"
+			"description_string"	"#Attrib_duckstreaks"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"positive"
+			"stored_as_integer"	"1"
+		}
+		"708"
+		{
+			"name"	"panic_attack"
+			"attribute_class"	"panic_attack"
+			"description_string"	"#Attrib_PanicAttack"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"positive"
+			"stored_as_integer"	"0"
+		}
+		"709"
+		{
+			"name"	"panic_attack_negative"
+			"attribute_class"	"panic_attack_negative"
+			"description_string"	"#Attrib_PanicAttackNegative"
+			"description_format"	"value_is_inverted_percentage" 
+			"hidden"	"0"
+			"effect_type"	"negative"
+			"stored_as_integer"	"0"
+		}
+		"710"
+		{
+			"name"	"auto fires full clip penalty"
+			"attribute_class"	"auto_fires_full_clip"
+			"description_string"	"#Attrib_AutoFiresFullClipNegative"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"negative"
+			"stored_as_integer"	"0"
+		}
+		"711"
+		{
+			"name"	"auto fires when full"
+			"attribute_class"	"auto_fires_when_full"
+			"description_string"	"#Attrib_AutoFiresWhenFull"
+			"description_format"	"value_is_additive"
+			"hidden"	"0"
+			"effect_type"	"negative"
+			"stored_as_integer"	"0"
+		}
+		"712"
+		{
+			"name"	"force weapon switch"
+			"attribute_class"	"force_weapon_switch"
+			"description_format"	"value_is_additive"
+			"hidden"	"1"
+			"effect_type"	"positive"
+		}
+		"719"
+		{
+			"name"	"weapon_uses_stattrak_module"
+			"attribute_class"	"weapon_uses_stattrak_module"
+			"attribute_type"	"string"
+			"description_format"	"value_is_additive"
+			"hidden"	"1"
+			"effect_type"	"strange"
+		}
+		"723"
+		{
+			"name"	"is_operation_pass"
+			"attribute_class"	"is_operation_pass"
+			"description_format"	"value_is_additive"
+			"hidden"	"1"
+		}
+		"724"
+		{
+			"name"	"weapon_stattrak_module_scale"
```

Comment:
It's just a text. It won't do anything without implementing attributes in the code.